### PR TITLE
NGP-3032 Updated conversations table

### PR DESCRIPTION
**Pull request by diegovillarroels**

### CODE REVIEW

File: src/components/Conversations/Conversations.tsx [hunk 1]
```diff
@@ -65,7 +72,48 @@ export default function Conversations() {
...
+            const jobPromises = listResults.map(async (job) => {
+                const healthSoapJobDetail = await getHealthScribeJob({
+                    MedicalScribeJobName: job.MedicalScribeJobName || '',
+                });
+
+                job.firstName = '';
+                if (healthSoapJobDetail.MedicalScribeJob && healthSoapJobDetail.MedicalScribeJob?.Tags) {
+                    job.firstName =
+                        healthSoapJobDetail.MedicalScribeJob?.Tags?.find((t) => t.Key === 'firstName')?.Value || '';
+                }
```
Comment (arieltrueforce) [inline src/components/Conversations/Conversations.tsx]: this piece of code is almost the same four times, if possible please try to use just one if statement and inside it made the assignations

**Review by arieltrueforce — APPROVED** [review]
Looks good to me just one improvement can be done

File: src/components/Conversations/Conversations.tsx [hunk 2]
```diff
@@ -65,7 +72,48 @@ export default function Conversations() {
                 return;
             }
 
-            const listResults: MedicalScribeJobSummary[] = listHealthScribeJobsRsp.MedicalScribeJobSummaries;
+            let listResults: MedicalSoapJobSummary[] = listHealthScribeJobsRsp.MedicalScribeJobSummaries;
+
+            const jobPromises = listResults.map(async (job) => {
+                const healthSoapJobDetail = await getHealthScribeJob({
```
Comment (araoz-nelson) [inline src/components/Conversations/Conversations.tsx:78]: Looks like there is no way to get the full list, right? as this is a POC I would say it is ok, but let's analyze performance if is required later.

File: src/components/Conversations/Conversations.tsx [hunk 3]
```diff
@@ -65,7 +72,48 @@ export default function Conversations() {
...
+                    MedicalScribeJobName: job.MedicalScribeJobName || '',
+                });
+
+                job.firstName = '';
+                if (healthSoapJobDetail.MedicalScribeJob && healthSoapJobDetail.MedicalScribeJob?.Tags) {
```
Comment (araoz-nelson) [inline src/components/Conversations/Conversations.tsx]: You can use this if conditional for all the fields, not required to create a separated for each one

File: src/components/Conversations/Conversations.tsx (hunk 1 above)
Comment (diegovillarroels) [reply src/components/Conversations/Conversations.tsx]: Done.

File: src/components/Conversations/Conversations.tsx (hunk 2 above)
Comment (diegovillarroels) [reply src/components/Conversations/Conversations.tsx:78]: Yes, one additional request is required to get all details.

File: src/components/Conversations/Conversations.tsx (hunk 3 above)
Comment (diegovillarroels) [reply src/components/Conversations/Conversations.tsx]: Done.

File: src/components/Conversations/conversationsPrefs.tsx [hunk 4]
```diff
@@ -52,23 +52,20 @@ export const DEFAULT_PREFERENCES: CollectionPreferencesProps.Preferences = {
     pageSize: 20,
     wrapLines: false,
     stripedRows: true,
-    visibleContent: ['MedicalScribeJobName', 'MedicalScribeJobStatus', 'CreationTime', 'ExpiresIn'],
+    //visibleContent: ['MedicalScribeJobName', 'MedicalScribeJobStatus', 'CreationTime', 'ExpiresIn'],
```
Comment (diegopraintree) [inline src/components/Conversations/conversationsPrefs.tsx]: Do we want to keep this commented code?

**Review by araoz-nelson — APPROVED** [review]

File: src/components/Conversations/conversationsPrefs.tsx (hunk 4 above)
Comment (diegovillarroels) [reply src/components/Conversations/conversationsPrefs.tsx]: Done.

**Review by arieltrueforce — APPROVED** [review]

**Review by diegopraintree — APPROVED** [review]

**Pull request merged**